ESPWebThingAdapter.h

```diff
@@ -167,6 +167,33 @@ class WebThingAdapter {
         prop["type"] = "string";
         break;
       }
+
+      prop["readOnly"] = property->readOnly;      
```

Comment:
Since properties default to readOnly false, I'd like this to be in `if (property->readOnly) {` to cut down on a bit of unnecessary serialization